### PR TITLE
Added installation docs

### PR DESCRIPTION
**Pull request by dgarcia360**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,10 +7,46 @@
...
+|-|-|-|
+| clientPrivateKey | REST client private key. | 000...000|
+| db.url | MongoDB [connection URL](https://github.com/nemtech/catapult-server/blob/master/resources/config-database.properties#L3). | mongodb://localhost:27017/ |
+| apiNode.host | API node connection host. | 127.0.0.1 |
+| apiNode.port | API node [connection port](https://github.com/nemtech/catapult-server/blob/master/resources/config-node.properties#L3). | 7900 |
+|api.publicKey | API node [public key](https://github.com/nemtech/catapult-server/blob/master/resources/config-user.properties#L4). | FFFF...FFF|
+| websocket.mq.host | ZeroMQ connection host. |  127.0.0.1 |
+| websocket.mq.port | ZeroMQ [connection port](https://github.com/nemtech/catapult-server/blob/master/resources/config-messaging.properties#L3). | 7902 |
+
+> **Note:** catapult-rest has to reach the API node, ZeroMQ and MongoDB ports. If you are running catapult-server on a VPS, you can bind the ports to your local development environment creating a **SSH tunnel**: ``ssh -L 27017:localhost:27017 -L 7900:localhost:7900 -L 7902:127.0.0.1:7902 -p 2357 <USER>@<VPS_IP>``
```
Comment (Vektrat) [inline README.md]: `an SSH tunnel` instead of `a SSH` the rest is fine for me, funny that i was just wondering/noticing why we had no README :p

Comment (dgarcia360) [reply README.md]: Well seen, fixed

File: README.md [hunk 2]
```diff
@@ -7,10 +7,46 @@
...
+|-|-|-|
+| clientPrivateKey | REST client private key. | 000...000|
+| db.url | MongoDB [connection URL](https://github.com/nemtech/catapult-server/blob/master/resources/config-database.properties#L3). | mongodb://localhost:27017/ |
+| apiNode.host | API node connection host. | 127.0.0.1 |
+| apiNode.port | API node [connection port](https://github.com/nemtech/catapult-server/blob/master/resources/config-node.properties#L3). | 7900 |
+|api.publicKey | API node [public key](https://github.com/nemtech/catapult-server/blob/master/resources/config-user.properties#L4). | FFFF...FFF|
+| websocket.mq.host | ZeroMQ connection host. |  127.0.0.1 |
+| websocket.mq.port | ZeroMQ [connection port](https://github.com/nemtech/catapult-server/blob/master/resources/config-messaging.properties#L3). | 7902 |
+
+> **Note:** catapult-rest has to reach the API node, ZeroMQ and MongoDB ports. If you are running catapult-server on a VPS, you can bind the ports to your local development environment creating an **SSH tunnel**: ``ssh -L 27017:localhost:27017 -L 7900:localhost:7900 -L 7902:127.0.0.1:7902 -p 2357 <USER>@<VPS_IP>``
```
Comment (rg911) [inline README.md]: @dgarcia360 `7902:127.0.0.1:7902` can be `7902:localhost:7902` to be consistent with other port forwarding.

Comment (dgarcia360) [reply README.md]: Thanks, solved!

**Pull request merged**